tests/ami/test_notification.py

```diff
@@ -280,6 +284,68 @@ async def test_admin_create_notification_no_registration(
     assert not httpx_mock.get_request()
 
 
+async def test_create_notification_partner_has_no_default_icon(
+    test_client: TestClient[Litestar],
+    db_session: AsyncSession,
+    user: User,
+    partner_auth: dict[str, str],
+    httpx_mock: HTTPXMock,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    partner = copy.deepcopy(partners["psl"])
+    partner.icon = ""
+    monkeypatch.setattr("app.auth.partners", {"psl": partner})
```

Comment:
I'm not sure I understand the use for this, isn't it [already an empty icon for PSL](https://github.com/numerique-gouv/ami-notifications-api/pull/354/files#diff-08c8106aaa9767a89653b6f98a018010f414d627880d6953d98f7c2f6852f1b9R17)?